package-list-aptget

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+echo "Setting things to install Bassa..."
```

Comment:
```suggestion
echo "Preparing to install Bassa..."
```

Reply:
@JaDogg bro I used `Setting things` as it will let the user know that this `echo` statement is about command that is going to install all required packages, So I found that more helpful :sweat_smile:

Reply:
I guess Preparing seems a good word @demon-36

Reply:
Can you change this to all scripts? @demon-36

Reply:
okay @vivonk will do it for all.